app/views/charge.njk

```diff
@@ -41,7 +41,7 @@
 
 
 {% block bodyStart %}
-  {% if (allowApplePay or allowGooglePay) and not savePaymentInstrumentToAgreement %}
+  {% if (allowApplePay or allowGooglePay) and not savePaymentInstrumentToAgreement and not moto%}
```

Comment:
Can you just fix the formatting to add a space before ` %}`

Reply:
thanks, done.